tsconfig.json

```diff
@@ -18,5 +18,6 @@
     "strict": true,
     "skipLibCheck": true,
     "strictNullChecks": true
-  }
+  },
+  "exclude": ["node_modules", "dist", "eslint.config.js", "prettier.config.js"]
 }
```

Comment:
Why these changes are needed?

Reply:
Does not work with `eslint.config.js` and ohter files.